src/main/java/com/metamx/metrics/SysMonitor.java

```diff
@@ -268,10 +273,16 @@ public void emit(ServiceEmitter emitter)
               log.error(e, "Failed to get FileSystemUsage[%s]", name);
             }
             if (fsu != null) {
-              final Map<String, Long> stats = ImmutableMap.of(
-                  "sys/fs/max", fsu.getTotal() * 1024,
-                  "sys/fs/used", fsu.getUsed() * 1024
-              );
+              final Map<String, Long> stats = ImmutableMap.<String, Long>builder()
+                  .put("sys/fs/max", fsu.getTotal() * 1024)
+                  .put("sys/fs/used", fsu.getTotal() * 1024)
+                  .put("sys/fs/files/count", fsu.getFiles())
+                  .put("sys/fs/files/free", fsu.getFreeFiles())
+                  .put("sys/fs/disk/read/count", fsu.getDiskReads())
+                  .put("sys/fs/disk/read/size", fsu.getDiskReadBytes())
```

Comment:
how are those metrics different than the existing `sys/disk/(read|write)/(size|count)`